python/hail/representation/__init__.py

```diff
@@ -12,4 +13,5 @@
            'Struct',
            'Call',
            'Pedigree',
-           'Trio']
+           'Trio',
+           'GenomeReference']
```

Comment:
I've always heard reference genome, like reference implementation. Wikipedia uses that: https://en.wikipedia.org/wiki/Reference_genome

And it looks to be more common:
https://books.google.com/ngrams/graph?content=genome+reference%2C+reference+genome&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cgenome%20reference%3B%2Cc0%3B.t1%3B%2Creference%20genome%3B%2Cc0